Changelog.md

```diff
@@ -1,5 +1,87 @@
 ## Gazebo Sim 7.x
 
+### Gazebo Sim 7.7.0 (2024-01-17)
+
+1. Allow using plugin file names and environment variables compatible with Garden and later
+    * [Pull request #2275](https://github.com/gazebosim/gz-sim/pull/2275)
+
+1. Added tutorial for Gazebo joint controller plugin
+    * [Pull request #2263](https://github.com/gazebosim/gz-sim/pull/2263)
+
+1. Revert to use of cos^2 in LiftDrag plugin
+    * [Pull request #2273](https://github.com/gazebosim/gz-sim/pull/2273)
+
+1. Default CMA in LiftDrag plugin to zero
+    * [Pull request #2272](https://github.com/gazebosim/gz-sim/pull/2272)
+
+1. Fix incorrect light direction in tunnel.sdf example
+    * [Pull request #2264](https://github.com/gazebosim/gz-sim/pull/2264)
+
+1. Fix DLL linkage/visibility issues
+    * [Pull request #2254](https://github.com/gazebosim/gz-sim/pull/2254)
+
+1. mecanum_drive: use mesh wheels in example world
+    * [Pull request #2250](https://github.com/gazebosim/gz-sim/pull/2250)
+
+1. environment_preload: fix windows compiler warnings
+    * [Pull request #2246](https://github.com/gazebosim/gz-sim/pull/2246)
+
+1. EnvironmentPreload: ignerr -> gzerr
+    * [Pull request #2245](https://github.com/gazebosim/gz-sim/pull/2245)
+
+1. Update friction parameters for skid steer example
+    * [Pull request #2235](https://github.com/gazebosim/gz-sim/pull/2235)
+
+1. Use sdf FindElement API to avoid const_cast
+    * [Pull request #2236](https://github.com/gazebosim/gz-sim/pull/2236)
+
+1. wind addition to advanced_lift_drag plugin
+    * [Pull request #2226](https://github.com/gazebosim/gz-sim/pull/2226)
```

Comment:
consider grouping with #2185

Reply:
Done in https://github.com/gazebosim/gz-sim/pull/2292/commits/0374146d0f641d0709f7633595d53fce761c3150